data/snippets.ts

```diff
@@ -528,6 +528,30 @@ const project = [
     name: "Project Spec",
     type: "template",
   },
+  {
+    id: nanoid(),
+    text: `gm :sparkles: :unicorn_face: :rainbow:
+
+*:arrow_backward: Last week:*
```

Comment:
Thomas had a good point about using emojis instead of their names (`:arrow_backward:`) so it can be used anywhere, not just Slack

WDYT?